Calendar/BookingHelpers/BookingServiceHelpers.ps1

```diff
@@ -0,0 +1,78 @@
+﻿
+function Get-GraphBookingsServices {
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$identity
+    )
+    $MBservices = Get-MgBookingBusinessService -BookingBusinessId $identity
+    $MBStaff = Get-MgBookingBusinessStaffMember -BookingBusinessId $identity
+    $services = @()
+    foreach ($service in $MBservices) {
+        #determine serviceType
+        $serviceType = ""
+        if ($service.StaffMemberIds.Count -gt 1) {
+            $serviceType = "N"
+        } else {
+            $serviceType = "1"
+        }
+        $serviceType+= ":"
+        if ($service.MaximumAttendeesCount -gt 1) {
+            $serviceType += "N"
+        } else {
+            $serviceType += "1"
+        }
+        #compose ServiceStaffIds
+        $serviceStaffIds = ""
+        foreach ($staffId in $service.StaffMemberIds) {
+            $serviceStaffIds += $staffId + "`r`n"
+        }
+        $staffName1 = ""
+        $staffName2 = ""
+        $staffName3 = ""
+        $staffName4 = ""
+        $staffName5 = ""
```

Comment:
Is 5 a Limit?

Reply:
No, but 5 is the number I use for name resolving from querying Staff table, if more than 5, need to look for the staffId's column